packages/eslint-plugin/src/rules/prefer-output-readonly.ts

```diff
@@ -19,17 +19,24 @@ export default createESLintRule<Options, MessageIds>({
     messages: {
       preferOutputReadonly:
         'Prefer to declare `@Output` as readonly since they are not supposed to be reassigned',
+      suggestFix: 'Add readonly modifier',
```

Comment:
I don't see any reason not to make the message key descriptive, it is what the eslint docs show in all their examples: https://eslint.org/docs/developer-guide/working-with-rules#providing-suggestions

...and allows for multiple suggestions to more easily be provided if applicable.

```suggestion
      addReadonlyModifier: 'Add readonly modifier',
```

Reply:
(Note the suggestion was just an easy way to add the feedback, it would need to be applied in all locations across multiple files)

Reply:
Good idea, I went with `suggestAddReadonlyModifier` after seeing many cases with the `suggest` prefix in `@typescript-eslint` repo as https://github.com/typescript-eslint/typescript-eslint/blob/1c1b572c3000d72cfe665b7afbada0ec415e7855/packages/eslint-plugin/src/rules/no-non-null-assertion.ts#L22.